src/bundle/Form/Locations/Actions.php

```diff
@@ -17,6 +18,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('delete', SubmitType::class)
+            ->add('add', SubmitType::class)
+            ->add('parentLocationId', TextType::class)
```

Comment:
As @bdunogier mentioned in the stand up could we not set the `data_class` of this to be `LocationCreateStruct` and this should avoid the call to `LocationService::newLocationCreateStruct`